### PR TITLE
replace "programming" with "learning" in the social rules

### PR DESCRIPTION
**Pull request by MaxBittker**
I thought that maybe since some workshops include other topics besides programming (right?)
this might be a helpful small modification of language. 
if you don't think so, feel free not to merge this, no explanation needed!

### CODE REVIEW

File: social-rules/index.html [hunk 1]
```diff
@@ -69,14 +69,14 @@ <h3 id="backseat-driving">No backseat driving</h3>
...
+        <p>There&rsquo;s another part of <i>no subtle -isms</i>: If you see racism, sexism, etc. outside of Ones and Zeros, please don&rsquo;t bring it in. For example, please don&rsquo;t start a discussion about the latest offensive comment from Random Tech Person Y. Everyone who comes to Ones and Zeros should have the same opportunity to focus on learning, and people from oppressed groups often find discussions of racism, sexism, etc. particularly hard to tune out. There are many places to discuss and debate these issues, but there are few where people can avoid them. Ones and Zeros is one of those places.</p>
```
Comment (MaxBittker) [inline social-rules/index.html:75]: this change was an accident and I don't know what it actually is :man_shrugging:

Comment (uniphil) [reply social-rules/index.html:75]: wait, I'm not sure I get this comment -- isn't this what the PR is about?

**Comment by uniphil** [comment]
yay yes! definitely an improvement -- we're not RC. thanks for this!

File: social-rules/index.html (hunk 1 above)
Comment (uniphil) [reply social-rules/index.html:75]: ooh I think maybe it's a github bug? looks like it's quoting a different part in the PR conversation view.

Comment (uniphil) [reply social-rules/index.html:75]: k checked it out on my local git: there is a programming -> learning change in three paragraphs including this one, but for some reason github isn't highlighting the words on this one, and generally seeming all kinds of weird 🤷

**Comment by uniphil** [comment]
Thank you Max!!! ❤️

File: social-rules/index.html (hunk 1 above)
Comment (MaxBittker) [reply social-rules/index.html:75]: ohhh, good catch!

**Pull request merged**